src/types/header.ts

```diff
@@ -0,0 +1,45 @@
+// Starknet
+import { BlockHeader } from "../deps.ts";
+
+// Eth
+import { bigIntToHex, Bloom, bytesToHex, JsonHeader } from "../deps.ts";
+
+const BLOCK_GAS_LIMIT = 30_000_000n;
+const BASE_FEE_PER_GAS = 100_000_000_000n;
+
+/**
+ * @param header - A Starknet block header.
+ * @param gasUsed - The total gas used in the block.
+ * @param logsBloom - The logs bloom of the block.
+ * @param receiptRoot - The transaction receipt trie root of the block.
+ * @param transactionRoot - The transaction trie root of the block.
+ * @returns The Ethereum block header in the json RPC format.
+ */
+export function toEthHeader(
+  { header, gasUsed, logsBloom, receiptRoot, transactionRoot }: {
+    header: BlockHeader;
+    gasUsed: bigint;
+    logsBloom: Bloom;
+    receiptRoot: Uint8Array;
+    transactionRoot: Uint8Array;
+  },
+): JsonHeader {
+  return {
+    parentHash: header.parentBlockHash,
+    uncleHash: "0x",
```

Comment:
is this standard for "emptyHash?"

Reply:
I used the default for rust's `Bytes` type. Not too sure this fits, but it felt incorrect to set the uncleHash to `0x0000....`.

Reply:
ok!

Reply:
checked on rust, default should be a 32 bytes long hex string, has been updated